migrations/versions/d01b87ff85d2_internationalize_bank_account_fields.py

```diff
@@ -0,0 +1,64 @@
+"""Internationalize bank account fields
+
+Revision ID: d01b87ff85d2
+Revises: 111954b830d9
+Create Date: 2020-03-03 22:29:42.800460
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'd01b87ff85d2'
+down_revision = '111954b830d9'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('bank_account', sa.Column('active', sa.Boolean(), nullable=True))
+    op.add_column('bank_account', sa.Column('address', sa.String(), server_default='19 Fleet Street, London EC4Y 1AA, UK', nullable=False))
+    op.add_column('bank_account', sa.Column('iban', sa.String(), server_default='BARCGB22', nullable=False))
+    op.add_column('bank_account', sa.Column('institution', sa.String(), server_default='Barclays Bank PLC', nullable=False))
+    op.add_column('bank_account', sa.Column('swift', sa.String(), server_default='BARCGB22', nullable=False))
+    op.alter_column('bank_account', 'acct_id',
+               existing_type=sa.VARCHAR(),
+               nullable=True)
+    op.alter_column('bank_account', 'sort_code',
+               existing_type=sa.VARCHAR(),
+               nullable=True)
+    op.create_index('ix_bank_account_currency_active', 'bank_account', ['currency', 'active'], unique=True)
+    # ### end Alembic commands ###
+
+    # manual custom operations #
+    op.execute("UPDATE bank_account SET active = true")
+    op.execute("UPDATE bank_account SET iban = CASE WHEN acct_id LIKE '%0526' THEN 'GB39BARC' ELSE 'GB21BARC' END || acct_id")
```

Comment:
:warning: Although these custom migration script steps have been tested for backwards-and-forwards compatibility in development (i.e. repeated `flask db upgrade` ... `flask db downgrade` both work and leave the DB and data in the expected state), this `UPDATE` operation on L35 assumes that only the two 'current' bank accounts exist when it runs.

Technically the script could perform an ISO-7064 checksum to figure out the IBAN check digs (the 'XX' in 'GBXXBARC'), but given only two existing accounts... it's easier simply to use a conditional check.